### ISSUE TITLE
No off-target

### ISSUE THREAD

**Issue by El84Ja** [lima1/PureCN#375]
Hi,
I have a baits exome probe BED file, tumor bam and vcf files . I used 8 unmatched normal sampoles to create my NormalDB.
I run all steps in default situation.
However, I think something is not correct with my interval files which results in unreasinable output for mu PureCN.
I have pasted the output for my different steps here:
Run INTERVAL step:
[Output .docx](https://github.com/user-attachments/files/16911580/Output.docx)
Thanks,

**Comment by lima1** [comment]
For whole exome you don't need off-target reads. Something is odd though. Do you have any baits outside chr1-22, X, Y in the BED file? It should work even if not, but worth trying without if any are there.

Can you confirm the unmatched normal samples are from exactly the same bait set?

**Comment by lima1** [comment]
Ill fix the deprecated warning for ggplot.

**Comment by El84Ja** [comment]
No, not any baits outside chr1-22, X and Y in the BED file. Yes, I confirm that unmatched normal samples are all precisely from the same baits set. Just know that all my vcf files has been created through Sentieon secondary analysis.
